src/components/views/settings/language.tsx

```diff
@@ -24,9 +24,9 @@ export const Language = () => {
       label: langData.name
     })
   );
-  const currentLanguage = languages.find(
-    ({value}) => value === i18n.language
-  ) || {value: 'en', label: 'English'};
+  const currentLanguage =
+    languages.find(({value}) => value === i18n.language) ||
+    languages.find(() => fallback === i18n.language);
```

Comment:
Shouldn't it be this:

```suggestion
    languages.find(({value}) => value === fallback);
```

Also have you tested your French phone with this change?

Reply:
That is a lot simpler, updated now